### ISSUE TITLE
Approval for submission?

### ISSUE THREAD

**Issue by petrelharp** [popgensims/manuscript#122]
Today was when we were nominally going to release this manuscript (which looks wonderful, by the way, thanks all!!). Are there any objections to polishing it off and preprinting this today? Major outstanding issues that shouldn't be postponed? The question is not "can it be improved", but "is it good enough", I think.

My suggestion is that we finish polishing and preprint today; then we can wait to submit until the new year if we want a bit more time. (I'm taking a careful pass through right now, and will address pending comments in #112.)

**Comment by asiepel** [comment]
I think it's very close but I just posted an issue about a small but important issue I'd like to address this afternoon.

**Comment by agladstein** [comment]
Sounds good. I'd like to get through the comments from https://github.com/popsim-consortium/manuscript/pull/112 and then I think it should be good.

**Comment by petrelharp** [comment]
Looks great!! I'm happy to sign off on this, and vote to incorporate final additions (eg #123; @agladstein comments) but to let @andrewkern preprint when he thinks appropriate.

**Comment by andrewkern** [comment]
Once this latest round of edits are done I'm going to take a pass through myself.

**Comment by petrelharp** [comment]
I'm good with adding Arabidopsis (or not if there are strong objections?).

**Comment by andrewkern** [comment]
I'm fixing a few minor typos but hoping for some more approvals here: @RyanGutenkunst @klohmueller

**Comment by klohmueller** [comment]
I approve! Thanks everyone!

**Comment by asiepel** [comment]
I had another quick look through and am happy for you to post when ready.  It's a nice introduction to the resource and a great first step to getting the word out.

**Comment by RyanGutenkunst** [comment]
I approve! Happy Holidays everyone!

**Comment by andrewkern** [comment]
We are submitted to bioRxiv! 🎉 

I suggest we wait on journal submission until after the break.

**Comment by asiepel** [comment]
Excellent.  Submission after the holiday sounds great.  Nice work everyone and enjoy the break!

**Comment by petrelharp** [comment]
Thanks, all - good work!
